snap7/exceptions.py

```diff
@@ -1,5 +1,18 @@
+import snap7
+
+
 class Snap7Exception(Exception):
     """
     A Snap7 specific exception.
     """
     pass
+
+
+class Snap7InvalidArea(Snap7Exception):
```

Comment:
I was trying to get rid of snap7 specific exceptions actually. In my opinion this is just a `ValueError`, and there is no need to create a new exception type.

Reply:
I didn't want to write the same error message 4 times, so I created an exception that keeps the error message.  If  `ValueError: X is not a valid area` is enough, I could rid of "try/catch" too.

Reply:
i think that is enough.

Reply:
btw, see discussion here https://github.com/gijzelaerr/python-snap7/issues/272